awscli/customizations/s3/results.py

```diff
@@ -268,6 +277,9 @@ def _record_progress_result(self, result, **kwargs):
         self._ongoing_progress[
             self._get_ongoing_dict_key(result)] += bytes_transferred
         self.bytes_transferred += bytes_transferred
+        if result.timestamp > self.start_time:
```

Comment:
In what scenario would you have `result.timestamp` be less than `self.start_time`?